app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<% provide :title, t(".signup") %>
+<h1><%= t ".signup" %></h1>
+<div class="row">
+  <div class="col-12">
+    <%= form_with(model: @user, local: true)  do |form| %>
```

Comment:
Sử dụng form tag để thực hiện tính năng này đi em? In ra parameter trong 2 trường hợp

Reply:
form_with: 
![Screenshot from 2020-07-31 13-32-16](https://user-images.githubusercontent.com/68731235/89007475-03df9a80-d333-11ea-9487-1c631f6b4816.png)
form_tag:
![Screenshot from 2020-07-31 13-36-51](https://user-images.githubusercontent.com/68731235/89007488-0b06a880-d333-11ea-9ef8-f1f51b988bb2.png)